srtcore/group.h

```diff
@@ -239,6 +239,19 @@ class CUDTGroup
     /// @retval false running link is unstable
     bool sendBackup_CheckRunningStability(const gli_t d, const time_point currtime);
     
+    /// Check link sending status
+    /// @param d              Group member iterator
+    /// @param currtime       Current time (logging only)
+    /// @param stat           Result of sending over the socket
+    /// @param lastseq        Last sent sequence number before the current sending operation
+    /// @param pktseq         Packet sequence number currently tried to be sent
+    /// @param w_u            CUDT unit of the current member (to allow calling overrideSndSeqNo)
```

Comment:
Mark all parameters with `w_` prefix `[OUT]` in doxy description, all others `[IN]`.

Reply:
Done